components/mdx/YoutubeEmbed.tsx

```diff
@@ -1,7 +1,14 @@
 export const YouTubeEmbed = ({ src }: { src: string }) => {
   return (
-    <div className="youtube-embed">
-      <iframe width="853" height="480" src={src} frameBorder="0" allowFullScreen />
+    <div className="relative pb-[56.25%] pt-[30px] h-0 2-full rounded-lg overflow-hidden my-[2em] border border-gray-200">
```

Comment:
If you're interested in maintaining a fixed aspect ratio like this, there is an official [aspect-ratio plugin](https://github.com/tailwindlabs/tailwindcss-aspect-ratio) for Tailwind that implements the bottom padding hack so you don't have to manually specify all of the necessary utility classes on the parent/child elements. 😀

Reply:
Fucking fancy!

Changed, thank you!!